src/client/voice/ClientVoiceManager.js

```diff
@@ -41,6 +75,27 @@ class ClientVoiceManager {
     return broadcast;
   }
 
+  /**
+   * Creates a voice adapter for a given channel, to be used by @discordjs/voice.
+   * @param {VoiceChannel} channel The voice channel to create an adapter for.
+   * @returns {DiscordGatewayAdapterCreator}
+   * @private
```

Comment:
why is this private?

Reply:
It is preferred to use `channel.createVoiceAdapter` instead.

Reply:
But the library is moving to structure methods mainly calling public methods on the Manager so users can do either way; this is similar to the Manager methods and there doesn't seem to be any disadvantage to users using this

Reply:
The code has been refactored to remove 2 declarations of this